Add 2.0 release post

moWerk opened the pull request:
Initial draft for the 2.0 announcement blog post.
Images are placeholders until better ones are made or found. 

Derived from [https://github.com/jrtberlin/aos2.0-post](https://github.com/jrtberlin/aos2.0-post)

moWerk committed:
Add AsteroidOS 2.0 blog post draft

moWerk committed:
Add AsteroidOS 2.0 blog post thumbnail placeholders

beroset commented on pages/news/2-0-release.md:
```diff
@@ -0,0 +1,194 @@
+- **Join our [community forums](https://asteroidos.org/community/)** to discuss ideas and share feedback.
+- **Report issues or suggest features** on our [GitHub repository](https://github.com/AsteroidOS/asteroid).
+- **Contribute to the codebase** by tackling open issues or developing new features.
+
```
We could mention translation as another way to contribute.

beroset reviewed (changes requested):
Looks great!  I especially love the pictures. I have a few suggestions and comments.

moWerk committed:
Remove thanks from listing of changes

eLtMosen commented on pages/news/2-0-release.md:
```diff
@@ -0,0 +1,194 @@
+Watchfaces are now paired with the user selected Wallpaper already in the Watchface gallery. Helping to find your favourite combination at a glance. Both pages received major performance improvements.
+   * [Nightstand mode](https://github.com/AsteroidOS/asteroid-settings/pull/49)  
+Use your watch as a bedside clock or simply show charging much more clearly. Selected watchfaces show a large charging status when power is connected. 
+The nightstand settings page makes this mode very versatile. Thanks to beroset for the initial idea and implementation.
```
Jepp, that was written before you joined the team even and copied over from that draft. I fully agree it is out of context here.

moWerk committed:
Change tyre cut-off to flat tyre shape

moWerk committed:
Add asteroid-weatherfetch and link to watchface GUI and IP connection

moWerk committed:
Add mention of asteroid-healt

moWerk committed:
Add Weblate translation to Getting Involved section

moWerk committed:
Change Documentation section to mention the MediaWiki

beroset reviewed (approved):
Looks good!

moWerk committed:
Add quickPanel mention and rename eLtMosen

Add QuickPanel description and rename eLtMosen to moWerk.

MagneFire commented:
This may be missing the new flatmesh stuff

FlorentRevest committed:
Misc 2.0 article fixes

- Fix up the list of watches
- Fix a typo
- Reword the experimental section
- Mention the new FlatMesh

MagneFire committed:
Bump release date

Signed-off-by: Darrel Griët <dgriet@gmail.com>

MagneFire committed:
Add rover

Signed-off-by: Darrel Griët <dgriet@gmail.com>

MagneFire committed:
Fix some typos

Signed-off-by: Darrel Griët <dgriet@gmail.com>

MagneFire committed:
Remove link to release page as it will not exist

Signed-off-by: Darrel Griët <dgriet@gmail.com>

MagneFire committed:
Fix some typos

Signed-off-by: Darrel Griët <dgriet@gmail.com>

moWerk committed:
Add video thumbnails for relevant sections

moWerk committed:
Add video thumbnails into ordered lists using html

FlorentRevest committed:
misc improvements to the 2.0 release article

Mainly
- Replaced the thumbnail pictures that were taken with a potato linux
  phone
- Added pictures for various sections
- Reordered sections for better visual consistency
- Fixed the publication date
- Fixed up the title
- Mentioned mainline Linux support

moWerk committed:
Swap square thumbnail for 2.0 rig picture
